content/pages/getting_started/what_should_i_make.md

```diff
@@ -14,32 +13,35 @@ These projects are great for beginners, no matter where your interests lie. Asid
 - Bottle openers
 - Nail header and nails
 
-## What next?
+Check out our [storyboards](/pages/curriculum/storyboards) page for step-by-step forged models of many of these projects to help you get started!
+
+# What next?
 Where do your interests lie? Your personal goals as a blacksmith should inform the projects you undertake.
 
-# Bladesmith Track
+## Bladesmith Track
 Swords are just long knives, and knives are just long leaves - start there:
 - Leaves
-- Blacksmith's/Camp knife
+- Blacksmith's knife
+- X-Rhea knife
```

Comment:
Add "knife-shaped objects"; basically mild steel dull knives. More forgiving to hammer out and "heat treat".